docs/api.md

```diff
@@ -0,0 +1,35 @@
+# ssh_scan API documentation
+
+**API Version:** 1.0
+
+This document explains the ssh_scan Web API, intended to be used in the Web API for ssh_scan.
```

Comment:
I'm wondering if at first we should have some label that states that this is a work in progress or a draft/design since the API doesn't exist yet